.gitignore

```diff
@@ -65,4 +65,8 @@ doc_sources_*
 
 # Python virtual environments
 venv/
-venv2/
\ No newline at end of file
+venv2/
+
+# Ignore the nd4j files that are created by javacpp at build to stop merge conflicts
+nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/nativeblas/Nd4jCpu.java
+nd4j/nd4j-backends/nd4j-backend-impls/nd4j-cuda/src/main/java/org/nd4j/nativeblas/Nd4jCuda.java
```

Comment:
It doesn't make sense to ignore files from `src/main`. Please propose something that makes sense.